source/index.html

```diff
@@ -1,12 +1,146 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html class = "page" lang="ru">
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Седона</title>
   </head>
-  <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
-
+  <body class="page__body">
+    <header class="main-header">
+      <a class="main-header__logo">
+        <img src="../images/logo_sedona.svg" width="185" height="96" alt="Седона">
+      </a>
+      <nav class="main-nav">
+        <ul class="main-nav__list site-list">
+          <li class="site-list__item site-list__item--active">
+            <a>
+              Главная
+            </a>
+          </li>
+          <li class="site-list__item">
+            <a href="#">
+              Фото и видео
+            </a>
+          </li>
+          <li class="site-list__item">
+            <a href="#">
+              Форма отзыва
+            </a>
+          </li>
+          <li class="site-list__item">
+            <a href="https://htmlacademy.ru/">
+              HTML Academy
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main-container">
+      <h1 class="main-container__title visually-hidden">Расскажем про Седону и причины посетить этот городок!</h1>
+      <img class="main-container__sedona-picture" src="" alt="Седона фотография на фон" width="1200" height="485">
+      <img class="main-container__welcome-picture" src="" alt="Добро пожаловать в Седону" width="458" height="352">
+      <section class="advantages">
+        <div class="advantages__wrapper">
+          <header class="advantages__header">
+            <h2 class="visually-hidden">
+              Причины посетить Седону
+            </h2>
+            <p class="advantages__slogan">
+              Седона — небольшой городок в Аризоне, заслуживающий большего!
+            </p>
+            <p class="advantages__intro">
+              Рассмотрим 5 причин, по которым Седона круче, чем Гранд Каньон!
+            </p>
+          </header>
+          <ol class="advantages__list">
+            <li class="advantages__item advantages__item--with-pic">
+              <div class="advantages__item-container">
+                <h3 class="advantages__title advantages__title--white">Настоящий городок</h3>
+                <p class="advantages__description advantages__description--white">Седона — не аттракцион для туристов, там течет своя жизнь</p>
+              </div>
+              <div class="advantages__image-container">
+                <img class="advantages__sedona-image" src="../images/sedona-2.jpg" alt="Седона город" width="800" height="384">
+              </div>
+              <ul class="advantages__promo-list promo-list">
+                <li class="promo-list__item">
+                  <span class="promo-list__logo--hostel"><span class="visually-hidden">Домик</span></span>
```

Comment:
Тут (и ниже) ошибка заключается в том, что модификатор указан у элемента, который еще не создан.
Мы будем прописывать общие стили для всех лого, и по отдельности ссылки на картинки, поэтому предлагаю вот так
```suggestion
                  <span class="promo-list__logo promo-list__logo--hostel"><span class="visually-hidden">Домик</span></span>
```